Source/Fuse.Reactive.Bindings/Each.uno

```diff
@@ -158,14 +158,11 @@ namespace Fuse.Reactive
 		*/
 		public object Items
 		{
-			get { return _items; }
+			get { return GetItems(); }
```

Comment:
Is this getter really an improvement? It just seems like churn to me...

Reply:
It's because I intended on removing `protected` and `internal` from `_items` to prevent direct access. Somehow that change got lost. I'll that back in -- the error is essentially caused by incorrect access to `_items`, that's why I'm hiding it now.

Reply:
Wait, wrong code, yes, I did make that change in the base class already.

Reply:
So no, it's not code churn, `return _items` can't work anymore.